mono/metadata/w32file-unix.c

```diff
@@ -2359,7 +2365,17 @@ CopyFile (const gunichar2 *name, const gunichar2 *dest_name, gboolean fail_if_ex
 		
 		return(FALSE);
 	}
-	
+
+#if HAVE_COPYFILE_H
```

Comment:
`#ifdef` to be consistent with https://github.com/mono/mono/pull/9038/files#diff-e3902264b6c6cb537e9eec9149668e83R17

Reply:
The top of the file uses #ifdef, the rest of the file uses #if.

Wont make this change.